docs/source/index.md

```diff
@@ -4,6 +4,10 @@
 
 For a quick introduction to the package, check out the [intro tutorial](../notebooks/intro_tutorial) which demonstrates the main functionality. For further details, check out the [code documentation](../documentation).
 
+Here's an example of how shading affects over the course of one day for a hexagonal layout:
```

Comment:
This sentence seems like it's missing something (...shading affects [something?])